### PR TITLE
Ben Wagners Bio Page

### LINKED ISSUE #24: Create Bio Page - Ben Wagner

### PR DESCRIPTION
**Pull request by benwagner813**
resolves #24

### CODE REVIEW

File: bios/team-three/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <h1>CSCI 2479 Spring 2022 &mdash; Bios</h1>
     <nav>
         <div><a href="./mihaela-valac.html">Mihaeal Valac</a></div>
-        <div><a href="../team-three/"></a></div>
+        <div><a href="./ben-wagner.html">Ben Wagner</a></div>
```
Comment (ryvrook) [inline bios/team-three/index.html:13]: Working link.

File: bios/team-three/ben-wagner.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <nav>
+          <div><a href="./index.html">Back to Team three</a></div>
+        </nav>
+        <div class="bio">
+            <h2>Ben Wagner</h2>
+            <ul class="qa">
+              <li><b>Where you are from originally?</b><br> Columbus, Ohio</li><br>
+              <li><b>Your major/track at CSCC?</b> <br> Web Development AAS</li><br>
+              <li>
+                <b> What you hope to do after school?</b> <br> I would like to find a relatively non-stressful web development job. Meaning a job that is within my capabilties yet I am still able to grow.
```
Comment (ryvrook) [inline bios/team-three/ben-wagner.html:20]: This is a good thing to look for. Remote might be of scope within what you are looking for.

File: bios/team-three/ben-wagner.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+              </li><br>
+              <li><b>What you enjoy doing as a hobby or pastime?</b> <br> I like to play PC games mainly DOTA 2. I also enjoy reading, mainly fantasy and sci-fi. More infrequently I like to go backpacking, I have been to many national parks and hope to visit more.
```
Comment (ryvrook) [inline bios/team-three/ben-wagner.html:22]: National Parks are beautiful, awesome!

**Review by DevVoxel — APPROVED** [review]
Good additions.  Nothing broken, and working links.  Good Bio!

**Pull request merged**